tests/ui_tools/test_popups.py

```diff
@@ -9,7 +9,7 @@
 from zulipterminal.api_types import Message
```

Comment:
> Is there an indication to use a ui: area somewhere? It's not unreasonable, but we've not included one in the recent past

No, it just felt natural to include it, so I added it. Thinking about it now, it might be a bit confusing to have an area be the same as a filename. I'll remove it for now.